### PR TITLE
Making `UntypedErrorInCatchRule` correctable

### PR DESCRIPTION
**Pull request by dirtydanee**
No issue link, just made the existing `UntypedErrorInCatchRule` correctable.

### CODE REVIEW

File: Source/SwiftLintFramework/Rules/UntypedErrorInCatchRule.swift [hunk 1]
```diff
@@ -49,17 +49,48 @@ public struct UntypedErrorInCatchRule: OptInRule, ConfigurationProviderRule {
...
+
+    fileprivate func violationRanges(in file: File) -> [NSRange] {
+        return file.match(pattern: type(of: self).regularExpression,
+                          with: [.keyword, .keyword, .identifier])
+    }
+}
+
+extension UntypedErrorInCatchRule: CorrectableRule {
+    public func correct(file: File) -> [Correction] {
+         let violations = violationRanges(in: file)
```
Comment (marcelofabri) [inline Source/SwiftLintFramework/Rules/UntypedErrorInCatchRule.swift]: indentation seems off

**Comment by marcelofabri** [comment]
Thanks! 🎉

**Comment by dirtydanee** [comment]
@jpsim good catch, it could lead to cases like that. 

I could update the auto correction only to cases where the variable is `let error`. That should not lead to compilation issues. What do you think?

**Comment by jpsim** [comment]
> I could update the auto correction only to cases where the variable is `let error`. That should not lead to compilation issues. What do you think?

Yes please!

**Pull request merged**